tests/dragonfly/cluster_test.py

```diff
@@ -252,14 +252,17 @@ def is_local_host(ip: str) -> bool:
     return True
 
 
-@dfly_args({"proactor_threads": 4, "cluster_mode": "emulated"})
+# --managed_service_info means that Dragonfly is running in a managed service, so some details
+# are hidden from users, see https://github.com/dragonflydb/dragonfly/issues/4173
+@dfly_args({"proactor_threads": 4, "cluster_mode": "emulated", "managed_service_info": "true"})
 async def test_emulated_cluster_with_replicas(df_factory):
-    master = df_factory.create(port=BASE_PORT)
+    master = df_factory.create(port=BASE_PORT, admin_port=BASE_PORT + 1000)
     replicas = [df_factory.create(port=BASE_PORT + i, logtostdout=True) for i in range(1, 3)]
 
     df_factory.start_all([master, *replicas])
 
     c_master = aioredis.Redis(port=master.port)
```

Comment:
please use `master.client()` and `master.admin_client()` instead of explicit `aioredis.Redis`. the latter handles the closing of the connections automatically.

Reply:
yeah, you get it built in.